.github/workflows/ci.yml

```diff
@@ -19,16 +19,30 @@ jobs:
   build:
     runs-on: ubuntu-18.04
     timeout-minutes: 30
-    env:      
+    env:
       MONGODB_VERSION: 3.6.9
+    strategy:
+      matrix:
+        include:
+          - name: Node 12
+            NODE_VERSION: 12.22.11
+          - name: Node 14
+            NODE_VERSION: 14.19.1
+          - name: Node 15
+            NODE_VERSION: 15.14.0
+          - name: Node 16
+            NODE_VERSION: 16.14.2
+          - name: Node 18
+            NODE_VERSION: 18.8.0
+      fail-fast: false
     steps:
     - name: Fix usage of insecure GitHub protocol
       run: sudo git config --system url."https://github".insteadOf "git://github"
     - uses: actions/checkout@v2
     - name: Use Node.js
       uses: actions/setup-node@v1
       with:
-        node-version: 14
+        node-version: ${{ matrix.node-version }}
```

Comment:
This should be `matrix.NODE_VERSION` if you look at the CI all the test run against `v16.18.0`.

Also should we worry about the tests running against package-lock.json v2?

Reply:
@dblythy Did you copy/paste this from another workflow file? Maybe it needs to be corrected there as well.

Reply:
@mtrezza @dblythy Should we add `npm install --no-save` to the CONTRIBUTION.md guide? I run npm > 7 and have to keep downgrading before my PR. This will prevent overriding `lockfileVersion: 2` in the `package-lock.json` from being commited.

https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile

Reply:
We didn't really have much of an issue with this. Whether the option is needed depends on the PR, so we'd need a whole paragraph devoted to explaining this. I doubt many read this anyway.

Reply:
Resolving as node ref is fixed